sequencer/src/block2.rs

```diff
@@ -688,7 +803,7 @@ mod tx_table_entry {
     }
 }
 
-type NsTable = <BlockPayload as hotshot::traits::BlockPayload>::Metadata;
+type NsTable = <Payload as hotshot::traits::BlockPayload>::Metadata;
```

Comment:
```suggestion
type NsTable = <Payload as BlockPayload>::Metadata;
```
Now that we are directly importing `BlockPayload` we should replace all occurrences of `hotshot::traits::BlockPayload` with `BlockPayload`.